worker/upgradeseries/upgrader_test.go

```diff
@@ -44,7 +44,7 @@ func (s *upgraderSuite) TestNotToSystemdCopyToolsOnly(c *gc.C) {
 
 	// No systemd file changes; just the new tools for the target series.
 	s.manager.EXPECT().CopyAgentBinary(
-		s.machineService, s.unitServices, paths.NixDataDir, "trusty", "precise", version.Current,
```

Comment:
Might as well remove the suite member and setup for this.

Reply:
OK